src/components/common/TmSessionImport.vue

```diff
@@ -52,10 +52,6 @@ import SessionFrame from "common/SessionFrame"
 import { mapGetters } from "vuex"
 import Steps from "../../ActionModal/components/Steps"
 
-const words24 = param => {
```

Comment:
have you considered words12 and words24?

Reply:
yes, I was actually thinking about it and making this an "official" PR 🤔 

Will go for it! Thanks 💪 !